### ISSUE TITLE
icons only on taskbar

### ISSUE THREAD

**Issue by shlozm** [bbidulock/icewm#633]
I would love to be able to have an option of 'icons only'  for windows on taskbar. Maybe the option is there but I can't find it.
Also, it would be nice to be able to configure/specify how wide an open window's icon+text is on the taskbar, the default behavior to assign about 1/3 of the task bar to each open window seems excessive and to my taste seems a bit awkward...

Thanks for a GREAT window-manager, I have been using it off and on for years! 
![screenshot](https://user-images.githubusercontent.com/96892473/147793073-9369a0cd-268f-43bb-bdf6-f600af83792d.png)

**Comment by shlozm** [comment]
OK, I figured out how to accomplish what I wanted. In the theme I set text color of all taskbar buttons to be the same as the button color, the text is now invisible. (Hovering over the button will show what the window is in a popup). In the preferences I set the TaskbarButtonWidthDivisor to 26 instead of the default 3
(BTW the memory usage on a fresh boot is in the 150s!)
Thanks again for a GREAT window manager!

![screenshot](https://user-images.githubusercontent.com/96892473/148563636-aaf401aa-f9a2-48d4-acd2-fcb7a51c4767.png)
.

**Comment by PPC-scripts** [comment]
Thanks for the proposed solutions for my suggested enhancements
- To make sure that the Taskbar does not show any text, making the text the same color as the background is doable (but not for the average user- it's not relevant on this forum, but I created a GUI to manage many of IceWM's settings from a single Control Panel like Window- and one of the options that has is precisely "Hide name of windows on the taskbar"- unfortunately I can't integrate changing the IceWN theme in that option).
- On displaying output of commands on system tray - I guessed that would be an heavy change. Thanks for suggesting yad: I'm a heavy yad user (my OS on all computers is antiX Linux and created some yad GUI's that were integrated into that OS. One of my first scripts was a GUI to manage quick-launch icons, making it easy to add or remove them, almost like in a dock) But, for some reason, at least in antiX Linux, under IceWM, yad system tray icon do not display tooltips (and they do on JWM, that's also included in antiX). So I'm not certain if my IceWM config file has some problems that stops it from displaying system tray yad tooltips (but display other system tray tooltips just fine).

Many thanks for the prompt reply!
